projects/plugins/jetpack/modules/masterbar/admin-menu/class-base-admin-menu.php

```diff
@@ -275,8 +275,11 @@ public function enqueue_scripts() {
 			'jetpack-admin-menu',
 			'jetpackAdminMenu',
 			array(
-				'upsellNudgeJitm'  => wp_create_nonce( 'upsell_nudge_jitm' ),
-				'jitmDismissNonce' => wp_create_nonce( 'jitm_dismiss' ),
+				'upsellNudgeJitm'       => wp_create_nonce( 'upsell_nudge_jitm' ),
+				'jitmDismissNonce'      => wp_create_nonce( 'jitm_dismiss' ),
+				'launchpadScreenOption' => get_option( 'launchpad_screen' ),
```

Comment:
Note: These changes make launchpad options available to the `admin-menu.js` script